utils/general.py

```diff
@@ -53,6 +54,25 @@ def download_file_blob_to_dir(directory, file_blob):
     os.remove(file_destination)
 
 
+def move_downloaded_data_to_single_directory(directories):
+
+    for directory in directories:
+        if not os.path.isdir(directory + "train"):
+            os.makedirs(directory + "train")
+        if not os.path.isdir(directory + "val"):
+            os.makedirs(directory + "val")
+
+        folders = os.listdir(directory)
+        for folder in folders:
+            if folder != "train" and folder != "val":
+                if folder.startswith("train"):
+                    copy_tree(directory + folder, directory + "train")
```

Comment:
Use a qualified import here. From [what I can tell](https://docs.python.org/3/library/shutil.html#shutil.copytree), shutil has a `copy_tree` already, so you don't need the extra import.

Reply:
Fixed 008c684370d75e8623bc1c74dfbc0b1f8a1dedf2